src/reqs.c

```diff
@@ -372,10 +372,13 @@ static struct request_s *process_request (struct conn_s *connptr,
                 if (reverse_url != NULL) {
                         safefree (url);
                         url = reverse_url;
-		} else if (config.reverseonly) {
+                } else if (config.reverseonly) {
+                        log_message (LOG_ERR, 
+                                    "Bad request, no mapping for '%s' found",
+                                    url);
```

Comment:
These need 1 space more of indentation: should be aligned not below the bracket but below the item in the bracket.